File: Timer.cs
using GTANetworkServer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

public class Time : Script
{
    public Time()
    {
        API.onUpdate += OnUpdateHandler;
    }

    public void OnUpdateHandler()
    {
        int hour = DateTime.Now.Hour;
        int minute = DateTime.Now.Minute;
        var date = DateTime.Now.ToString("dd.MM.yyyy");
        var time = DateTime.Now.ToString("HH:mm");
        API.setTime(hour, minute);
        foreach (Client player in API.getAllPlayers())
        {
            API.setEntitySyncedData(player, "dw_date", date);
            API.setEntitySyncedData(player, "dw_hour", time);
        }
    }
}

File: timer.js
var res_X = API.getScreenResolutionMantainRatio().Width;
var res_Y = API.getScreenResolutionMantainRatio().Height;
API.onUpdate.connect(function () {
var phandle = API.getLocalPlayer();
var date = API.getEntitySyncedData(phandle, "dw_date");
var hour = API.getEntitySyncedData(phandle, "dw_hour");
if(date !== null && hour !== null) {
API.drawText(`${date}`, res_X - 150, res_Y - 1035, 0.5, 255, 255, 255, 255, 7, 1, false, true, 0);
API.drawText(`${hour}`, res_X - 150, res_Y - 1015, 1, 255, 255, 255, 255, 7, 1, false, true, 0);
}
});